go.mod

```diff
@@ -253,6 +253,7 @@ require (
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.opentelemetry.io/otel/metric v1.20.0 // indirect
+	go.opentelemetry.io/otel/sdk v1.20.0
```

Comment:
Are we able to remove the dependency on the OpenTelemetry SDK? Ideally we would only reference the trace API

Reply:
yes, a previous iteration called on the otel span event type directly (which lives in the otelsdk pkg), but that's since been removed, so I'll clear this unused package from the go.mod

Reply:
Great, thanks!